src/main/java/com/google/jenkins/plugins/computeengine/ComputeEngineCloud.java

```diff
@@ -181,8 +203,25 @@ public String getProjectId() {
         return projectId;
     }
 
-    public void addConfiguration(InstanceConfiguration config) {
-        configurations.add(config);
+    public List<InstanceConfiguration> getConfigurations() {
```

Comment:
Missing javadoc header for method. (FYI, we're going to be collectively trying to address tech debt like this going forward, so as code gets touched we'll call it out).